FNPlugin/Collectors/SolarWindCollector.cs

```diff
@@ -141,41 +156,63 @@ public override void OnFixedUpdate()
             }
         }
 
```

Comment:
Yes, I was talking about the function below and I didn't create it myself , I got it from a temporary fix in Galactic neighborhood which used it to calculate solar flux